### PR TITLE
ODH 0.9.0 Release

### PR DESCRIPTION
**Pull request by LaVLaS**
CSV release for odh 0.9.0

Closes ! #73

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,151 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+
```
Comment (vpavlin) [inline deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml:7]: Is the example empty on purpose?

Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml]: No. I updated it to include the [kfdef/kfctl_openshift.yaml](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift.yaml)

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide as complete a solution as possible for both hobby and enterprise users.
+
+    ### Components
+    * Open Data Hub Dashboard - ***NEW*** A webui dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support. Notebook with support Elyra and JupyterLab
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml:233]: Can we replace "webui" with web portal? or just web dashboard?

**Comment by vpavlin** [comment]
Looks good when @nakfour's comment is addressed.

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml (hunk 2 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml:233]: Updated to say `web dashboard`

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.7.1 - Monitoring and alerting tool
+    * Grafana v7.1.1 - Data visualization and monitoring
+    * Airflow v1.10.2 - Workflow management
+    * AI-Library v0.6.0 - An open source collection of machine learning algorithms
+    * Seldon v1.2.0 - Open source platform for deploying machine learning models
+    * Argo v2.7.0 - Container-native Workflow Engine
+    * Apache Superset v0.34.0 - Open source application for data exploration and visualization
+    * Apache Kafka v2.4.0 - The open source stream processing platform
```
[the remark this answers is not in the document]
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml:242]: Updated to `v0.30.0`. It looks like the operator version matches the version of superset python package.

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide as complete a solution as possible for both hobby and enterprise users.
+
+    ### Components
+    * Open Data Hub Dashboard - ***NEW*** A webui dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support. Notebook with support Elyra and JupyterLab
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.7.1 - Monitoring and alerting tool
+    * Grafana v7.1.1 - Data visualization and monitoring
+    * Airflow v1.10.2 - Workflow management
```
[the remark this answers is not in the document]
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml:237]: The current versions listed are for the application version.

I confirmed that 
Grafana v7.1.1 is deployed
I updated the Prometheus version to v2.16.0 which is reflected in the Prometheus build version

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,347 @@
...
+    As part of our work to extend the capabilities of Kubeflow on OpenShift, we have KfDef manifests that combine components of Open Data Hub and Kubeflow into a single deployment. You can find more information about how we combine these components on [opendatahub.io](http://opendatahub.io/docs/kubeflow/mixing.html)
+
+    ### Available Channels
+
+    #### Beta
+
+    Channel `beta` offers the latest version of Open Data Hub operator. It is based on Kubeflow Operator and Kustomize for deployment configuration. This version is compatible with [Open Data Hub manifests](https://github.com/opendatahub-io/odh-manifests/) as well as [Kubeflow manifests](https://github.com/opendatahub-io/manifests). This channel will offer new components and features.
+
+    To deploy Kubeflow, use [this example](https://github.com/opendatahub-io/manifests/blob/v1.0-branch-openshift/kfdef/kfctl_openshift.yaml).
+
```
[the remark this answers is not in the document]
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml:255]: Updated

File: deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml]: I moved elyra to it's own bullet with text
```
* Elyra v0.0.5 - JupyterLab notebooks with support for AI workflows
```

**Review by crobby — APPROVED** [review]
Looks right to me

**Pull request merged**